bin/init_repo.sh

```diff
@@ -37,12 +40,21 @@ for template_file in "${template_files[@]}" ; do
         for idx in "${!user_values[@]}" ; do
             placeholder=${placeholders[idx]}
             placeholder_name="${placeholder%%:*}"
-            result=`echo "$result" | sed "s/{{$placeholder_name}}/${user_values[idx]}/g"`
+            result=`echo "$result" | sed "s#{{$placeholder_name}}#${user_values[idx]}#g"`
+            if [ "$placeholder_name" = "SRC" ] && [ ${user_values[idx]} != 'src' ]; then
```

Comment:
Well, this specific placeholder is actually different from other ones since this does not act only on the setup.cfg file but also renames a folder. So, even if I do not like special behaviours either, I think that this specific case is actually an exception to the general behaviour of other placeholders.